FreeRTOS-Plus/Demo/FreeRTOS_Cellular_Interface_Windows_Simulator/Common/cellular_setup.c

```diff
@@ -189,19 +215,51 @@ bool setupCellular( void )
     if( cellularStatus == CELLULAR_SUCCESS )
```

Comment:
After registration, you may need to wait for an OMA-DM or LWM2M transaction to configure the PDP contexts / bearer profiles. This auto-configuration often overwrites the default configuration for some carriers in North America.

Reply:
The setup example code assume the primary EPS bearer is used for internet access.
If internet access do not use the primary EPS bearer and need extra configuration to activate it, developers need to modify the setup sequence to enable internet access.
We may add the code and demo configuration for this feature if it is required.

Reply:
A demo showing secondary pdp context activation would be nice at some point in the future though.